app/src/main/java/com/heig/atmanager/HomeFragment.java

```diff
@@ -0,0 +1,152 @@
+    private RecyclerView goalsRecyclerView;
+    private RecyclerView.Adapter goalsAdapter;
+    private RecyclerView.LayoutManager goalslayoutManager;
+
+    // Task feed
+    ArrayList<Task> tasks; // user data
+    private RecyclerView tasksRecyclerView;
+    private RecyclerView.Adapter tasksAdapter;
+    private RecyclerView.LayoutManager taskslayoutManager;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View v = inflater.inflate(R.layout.fragment_home, container, false);
+
+        // ----------- TEMP -----------
+        goals = new ArrayList<>();
+        goals.add(new Goal("SQUATS", 100, 70));
+        goals.add(new Goal("KMS", 8, 3));
+        tasks = new ArrayList<>();
+        tasks.add(new Task("Task1"));
+        tasks.add(new Task("Task2"));
+        tasks.add(new Task("Task3"));
+        tasks.add(new Task("Task4"));
+        // --------- END TEMP ---------
+
+        // Greeting
+        greetingText = (TextView) v.findViewById(R.id.greeting_text);
+        greetingText.setText(getGreetings());
+
+        // Task feed
+        setupTasksRecyclerView(v, tasks);
+
+        // Goal feed
+        setupGoalsRecyclerView(v, goals);
+
+        return v;
+    }
+
+    private void setupTasksRecyclerView(View v, ArrayList<Task> tasks) {
+
+        tasksRecyclerView = (RecyclerView) v.findViewById(R.id.tasks_rv);
+
+        // use this setting to improve performance if you know that changes
+        // in content do not change the layout size of the RecyclerView
+        tasksRecyclerView.setHasFixedSize(true);
+
+        // use a linear layout manager
+        taskslayoutManager = new LinearLayoutManager(v.getContext());
+        tasksRecyclerView.setLayoutManager(taskslayoutManager);
+
+        // specify an adapter (see also next example)
+        tasksAdapter = new TaskFeedAdapter(tasks);
+        tasksRecyclerView.setAdapter(tasksAdapter);
+    }
+
+    private void setupGoalsRecyclerView(View v, ArrayList<Goal> goals) {
+
+        goalsRecyclerView = (RecyclerView) v.findViewById(R.id.goals_rv);
+
+        // use this setting to improve performance if you know that changes
+        // in content do not change the layout size of the RecyclerView
+        goalsRecyclerView.setHasFixedSize(true);
+
+        // use a (horizontal) linear layout manager
+        goalslayoutManager = new LinearLayoutManager(v.getContext(),
+                LinearLayoutManager.HORIZONTAL, false);
+        goalsRecyclerView.setLayoutManager(goalslayoutManager);
+
+        // specify an adapter (see also next example)
+        goalsAdapter = new GoalFeedAdapter(goals);
+        goalsRecyclerView.setAdapter(goalsAdapter);
+    }
+
+    /**
+     * Get the welcoming sentence (dynamic with user's data)
+     * @return the proper greetings
+     */
+    private String getGreetings() {
+        Calendar rightNow = Calendar.getInstance();
```

Comment:
Calendar is a singleton, it return a calendar object. You fetch hours after the getting the instance, the variable name is not correct
rightNow => calendar